### PR TITLE
Skill :  Spur Skills

### PR DESCRIPTION
**Pull request by AliScrooge**
Added Spur Skills : Adjacent allies gain +4 to their respective stat.

### CODE REVIEW

**Comment by MokhaLeee** [comment]
mind the your code style and indentation issues

File: Wizardry/Core/BattleSys/Source/PreBattleCalc.c [hunk 1]
```diff
@@ -653,10 +652,48 @@ void PreBattleCalcAuraEffect(struct BattleUnit * attacker, struct BattleUnit * d
...
+		{
+			if(!IsMagicAttack(attacker))
+                    	attacker->battleAttack  += 4;
+		}
+#endif
+
+#if (defined(SID_SpurMag) && (SID_SpurMag < MAX_SKILL_NUM))
+                if (SkillTester(unit, SID_SpurMag) && range1[i] == 1)
+		{
+			if(IsMagicAttack(attacker))
```
Comment (MokhaLeee) [inline Wizardry/Core/BattleSys/Source/PreBattleCalc.c:669]: Use four space rather than one TAB on indentation.

Comment (AliScrooge) [reply Wizardry/Core/BattleSys/Source/PreBattleCalc.c]: Should be good now. Sorry for the disagreement, I had now idea Github handled tabulation differently than 4 space

**Comment by MokhaLeee** [comment]
https://github.com/MokhaLeee/fe8u-cskillsys-kernel/commit/76328b2992c05f98a3d23f1658bc60e92398fb2e

**Comment by MokhaLeee** [comment]
https://github.com/MokhaLeee/fe8u-cskillsys-kernel/commit/b5ea8898ec2a3843021ef7f47b686d80f171896f

**Pull request merged**